README.md

```diff
@@ -64,8 +63,8 @@ This project is used to manage IAM permissions for
 |------|-------------|------|---------|:--------:|
 | assume_access_dm_terraform_backend_policy_description | The description to associate with the IAM policy that allows assumption of the role that allows access to Domain Manager-related Terraform backend resources. | `string` | `The IAM policy that allows assumption of the role that allows access to Domain Manager-related Terraform backend resources.` | no |
 | assume_access_dm_terraform_backend_policy_name | The name to assign the IAM policy that allows assumption of the role that allows access to Domain Manager-related Terraform backend resources. | `string` | `AssumeAccessDomainManagerTerraformBackend` | no |
-| assume_provisiondomainmanager_policy_description | The description to associate with the IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the User Services accounts. | `string` | `The IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the User Services accounts.` | no |
-| assume_provisiondomainmanager_policy_name | The name to assign the IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the User Services accounts. | `string` | `UserServices-AssumeProvisionDomainManager` | no |
+| assume_domainmanager_provisionaccount_policy_description | The description to associate with the IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the Domain Manager accounts. | `string` | `The IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the Domain Manager accounts.` | no |
+| assume_domainmanager_provisionaccount_policy_name | The name to assign the IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the Domain Manager accounts. | `string` | `DomainManager-AssumeProvisionAccount` | no |
```

Comment:
Should "accounts" be singular here?

Reply:
I pluralized it here because this policy allows assumption of the ProvisionAccount roles in both the Staging and Production Domain Manager accounts.

Reply:
Ah, thanks for the explanation!